nextcord/guild.py

```diff
@@ -591,6 +591,14 @@ def large(self) -> bool:
                 return len(self._members) >= 250
         return self._large
 
+    @property
+    def invites_paused(self) -> bool:
+        """:class:`bool`: Indicates if the guild's invites are paused.
+
+        .. versionadded:: 2.4
+        """
+        return "INVITES_DISABLED" in self.features.copy()
```

Comment:
Why is a copy of the features made? This is a read-only operation after all.

Reply:
https://github.com/nextcord/nextcord/pull/932/commits/68495ece366018adc4a7b4cd98cffa103b8b4f49